Cargo.toml

```diff
@@ -21,6 +21,7 @@ libtock_platform = { path = "platform" }
 libtock_proximity = { path = "apis/proximity" }
 libtock_runtime = { path = "runtime" }
 libtock_temperature = { path = "apis/temperature" }
+libtock_ambient_light = { path = "apis/ambient_light" }
```

Comment:
Please put this into alphabetical order (between `libtock_alarm` and `libtock_buttons`), both here and in `members`.

Reply:
Everything should be in the right order now! Thanks for the review!